app/templates/src/main/webapp/scripts/app/account/password/_password.directive.js

```diff
@@ -7,7 +7,7 @@ angular.module('<%=angularAppName%>')
             replace: true,
             restrict: 'E',
             template: '<div id="strength">' +
-                '<small translate="global.messages.validate.newpassword.strength">Password strength:</small>' +
+                '<small<% if(enableTranslation){ %> translate="global.messages.validate.newpassword.strength"<% } %>>Password strength:</small>' +
```

Comment:
even if we let this sit in here its still valid html and will not cause any issues, that is why I left this as such originally